dev/README.md

```diff
@@ -83,19 +83,16 @@ and/or chef-server-ctl commands.
 
 * Changes to erchef erlang files will be picked up and recompiled
   automatically shortly after you save them on the host.
-* To test cookbook changes, load them with `dvm load omnibus private-chef-cookbooks`.
-  Then run `chef-server-ctl reconfigure` in the VM to pick up the changes.
-* upgrades and chef-server-ctl command changes/additions will be
-  available very quickly after you save them on the host (< 5 seconds)
+* To test cookbook changes, load them with `dvm load omnibus server-ctl-cookbooks`
+  run `chef-server-ctl reconfigure` in the VM as needed to pick up the changes.
```

Comment:
It is looking like `dvm load omnibus server-ctl-cookbooks` is also triggering the `chef-server-ctl reconfigure`.